Gems/EMotionFX/Code/EMotionFX/Pipeline/SceneAPIExt/Groups/ActorGroup.cpp

```diff
@@ -76,6 +80,27 @@ namespace EMotionFX
                 m_selectedRootBone = selectedRootBone;
             }
 
+            void ActorGroup::SetBestMatchingRootBone(const AZ::SceneAPI::Containers::SceneGraph& sceneGraph)
+            {
+                auto nameContentView = AZ::SceneAPI::Containers::Views::MakePairView(sceneGraph.GetNameStorage(), sceneGraph.GetContentStorage());
+                auto graphDownwardsView = AZ::SceneAPI::Containers::Views::MakeSceneGraphDownwardsView<AZ::SceneAPI::Containers::Views::BreadthFirst>(
+                    sceneGraph, sceneGraph.GetRoot(), nameContentView.begin(), true);
+
+                for (auto it = graphDownwardsView.begin(); it != graphDownwardsView.end(); ++it)
+                {
+                    if (!it->second)
```

Comment:
a comment here why "it->second" is not a good root node would help here

Reply:
Embedded the check into the line below to make it more clear, this is just skipping nullptrs.